admin/app/components/solidus_admin/orders/show/component.html.erb

```diff
@@ -16,8 +16,8 @@
     <%= page_with_sidebar_aside do %>
       <%= render component('ui/panel').new(title: panel_title_with_more_links(t(".customer"), [
         link_to(t(".edit_email"), "#", class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
-        link_to(t(".edit_shipping"), "#", class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
-        link_to(t(".edit_billing"), "#", class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
+        link_to(t(".edit_shipping"), solidus_admin.new_order_ship_address_path(@order), class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
+        link_to(t(".edit_billing"), solidus_admin.new_order_bill_address_path(@order), class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
```

Comment:
optional nit: squash this commit to the one touching routes, possibly to the one adding the controller